time/time.go

```diff
@@ -0,0 +1,75 @@
+package time
+
+import (
+	"fmt"
+	"time"
+)
+
+// Time is a wrapper around time.Time that supports JSON marshaling.
+type Time time.Time
+
+func (t *Time) String() string {
+	if t == nil || time.Time(*t).IsZero() {
+		return ""
+	}
+
+	return time.Time(*t).Format(time.RFC3339Nano)
+}
+
+func (t *Time) MarshalJSON() ([]byte, error) {
+	if t == nil {
+		return []byte(""), nil
```

Comment:
Shouldn't this be null?
```suggestion
		return []byte("null"), nil
```